app/models/moves/basic.rb

```diff
@@ -44,6 +44,19 @@ module Moves
   class Basic < Move
     include ::Rollable
 
+    def self.default_moves
```

Comment:
If you have a class method with an array, a great alternative is a constant on the class.

Reply:
But we can get rid of this method.